jenkins/job-configs/kubernetes-jenkins/kubernetes-e2e-gce.yaml

```diff
@@ -596,7 +563,38 @@
                 export JENKINS_PUBLISHED_VERSION="ci/latest-1.4"
                 export GINKGO_TEST_ARGS="--ginkgo.focus=\[Feature:(externalTrafficLocalOnly|PetSet)\]|ScheduledJob"
                 export PROJECT="k8s-jkns-e2e-gce-alpha1-4"
-        # TODO: Move gce-scalability-release-1.3 here (gce-scalability-release-1.4) once we cut the release branch
+        - 'gce-scalability-release-1.4':  # kubernetes-e2e-gce-scalability-release-1.4
+            description: 'Run scalability E2E tests on GCE from the release-1.4 branch.'
+            timeout: 120
+            cron-string: '@daily'
+            job-env: |
+                export JENKINS_PUBLISHED_VERSION="ci/latest-1.3"
+                export E2E_NAME="e2e-scalability-1-3"
+                export GINKGO_TEST_ARGS="--ginkgo.focus=\[Feature:Performance\] \
+                                         --gather-resource-usage=true \
+                                         --gather-metrics-at-teardown=true \
+                                         --gather-logs-sizes=true \
+                                         --output-print-type=json"
+                # Use the 1.1 project for now, since it has quota.
+                # TODO: create a project k8s-e2e-gce-scalability-release and move this test there
+                export PROJECT="k8s-e2e-gce-scalability-1-1"
+                export FAIL_ON_GCP_RESOURCE_LEAK="false"
+                # Override GCE defaults.
+                export KUBE_GCE_ZONE="us-east1-b"
+                export MASTER_SIZE="n1-standard-4"
+                export NODE_SIZE="n1-standard-1"
+                export NODE_DISK_SIZE="50GB"
+                export NUM_NODES="100"
+                export REGISTER_MASTER="true"
+                # Reduce logs verbosity
+                export TEST_CLUSTER_LOG_LEVEL="--v=2"
+                # TODO: Remove when we figure out the reason for occasional failures #19048
+                export KUBELET_TEST_LOG_LEVEL="--v=4"
+                # Increase resync period to simulate production
+                export TEST_CLUSTER_RESYNC_PERIOD="--min-resync-period=12h"
+                # Increase delete collection parallelism
+                export TEST_CLUSTER_DELETE_COLLECTION_WORKERS="--delete-collection-workers=16"
+                export KUBE_NODE_OS_DISTRIBUTION="debian"
```

Comment:
I don't know what quota we need for a new scalability project. if someone has that info handy, then sure.

(we could then change the release scalability project name to -release rather than -1-1 too.)